### PR TITLE
fix example in set selector

### PR DESCRIPTION
**Pull request by xilabao**
example run failed

### CODE REVIEW

**Comment by resouer** [comment]
@k8s-bot ok to tes

**Comment by resouer** [comment]
@k8s-bot ok to test

File: pkg/kubectl/cmd/set/set_selector.go [hunk 1]
```diff
@@ -65,7 +65,7 @@ var (
         Note: currently selectors can only be set on Service objects.`)
 	selectorExample = templates.Examples(`
         # set the labels and selector before creating a deployment/service pair.
-        kubectl create service clusterip my-svc -o yaml --dry-run | kubectl set selector --local -f - 'environment=qa' -o yaml | kubectl create -f -
+        kubectl create service clusterip my-svc --clusterip="None" -o yaml --dry-run | kubectl set selector --local -f - 'environment=qa' -o yaml | kubectl create -f -
```
Comment (resouer) [inline pkg/kubectl/cmd/set/set_selector.go]: In what release this example fail? Latest master branch?

Comment (xilabao) [inline pkg/kubectl/cmd/set/set_selector.go]: @resouer In latest master branch

**Pull request merged**